### PR TITLE
Add bumpver as release tool

### PR DESCRIPTION
**Pull request by unkcpz**

### CODE REVIEW

File: setup.cfg [hunk 1]
```diff
@@ -1,6 +1,6 @@
 [metadata]
 name = aiidalab_mfa_cscs
-version = 0.1-alpha
```
Comment (yakutovicha) [inline setup.cfg:3]: The 0.1-alpha was good. You can change it to `0.1.0a1` also

**Review by yakutovicha — CHANGES_REQUESTED** [review]
Cannot say anything against bumpver. Just change the version back to alpha, the rest is good 👍

File: setup.cfg (hunk 1 above)
Comment (unkcpz) [reply setup.cfg:3]: This is not an actual release, after merge I'll run bumpver to make release of `0.1.0a0`. I set it to 0.0.1 so I can bump it later to higher version, if start with `0.1.0a0` then it has to release `0.1.0a1`. But should be not a big problem. Do you think still need to change?

Comment (unkcpz) [reply setup.cfg:3]: Anyway I change it to `v0.1.0a0` ;)

Comment (yakutovicha) [reply setup.cfg:3]: You can release `v0.1.0` also starting from the alpha release.

Also, we don't need to release alpha's. If the version is `0.x.x` we are allowed to do breaking changes. This is what the `semantic` version says.

Which brings me to another question: do we want to have it semantic? Why don't we do it the date based (same as QE app)?

Comment (unkcpz) [reply setup.cfg:3]: I don't know. I prefer the version of aiidalab-launch `v2023.1018` in format `version_pattern = "vYYYY.BUILD[PYTAGNUM]"`

Comment (yakutovicha) [reply setup.cfg:3]: > I don't know. I prefer the version of aiidalab-launch `v2023.1018` in format `version_pattern = "vYYYY.BUILD[PYTAGNUM]"`

Maybe use this schema then, I am ok with that!

Comment (unkcpz) [reply setup.cfg:3]: Voila, it is "v2023.0000" and after run `bumpver` it will be "v2023.1001".

**Review by yakutovicha — APPROVED** [review]
Looks good!

**Pull request merged**